.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Also add `.next` folder

Reply:
I have removed the .next folder and added it to .gitignore.